### PR TITLE
[W4.7][W11-A2]Zhan Yu

### PR DESCRIPTION
**Pull request by ZY-Ang**
Rebased LO-Encapsulation edits to org head

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -33,4 +33,12 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /**
+     * Getter method for feedbackToUser string
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/CommandResult.java]: please check the coding standards for the format of header comments :-)

**Comment by okkhoy** [comment]
@ZY-Ang good job on updating the tests as well in the exercise!
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/commands/CommandResult.java (hunk 1 above)
Comment (ZY-Ang) [reply src/seedu/addressbook/commands/CommandResult.java]: (Y) 19d813d6716fb586b7d1838ca8d01c44e4ffeaa9
```In method header comments, the first sentence should start in the form Returns …​, Sends …​, Adds …​ (not Return or Returnning etc.)```

**Comment by ZY-Ang** [comment]
@okkhoy regarding closing PRs, does this apply to my other PRs not yet marked as Reviewed in week 4?

**Comment by okkhoy** [comment]
@ZY-Ang only this one! i will spam you with "close the PR" for every PR :-P

**Pull request closed without merging**